### PR TITLE
Fix math docstrings in `staticKS.py`

### PR DESCRIPTION
**Pull request by timcallow**
Corrects errors in TeX typsetting in docstrings of `staticKS.py` module

### CODE REVIEW

File: atoMEC/staticKS.py [hunk 1]
```diff
@@ -808,8 +825,10 @@ def calc_E_ha(density, xgrid):
         Notes
         -----
         The Hartree energy is given by
-        .. math:: E_{ha} = 2\pi \int\dd{r}_0^{r_s} r^2 n(r) v_{ha}(r),
-        where :math: 'v_{ha}(r)' is the Hartree potential
+
+        .. math:: E_\mathrm{ha} = 2\pi\int_0^{r_s}\mathrm{d}r r^2 n(r) v_\mathrm{ha}(r),
+
+        where :math:`v_{ha}(r)` is the Hartree potential
```
Comment (DanielKotik) [inline atoMEC/staticKS.py:831]:
```suggestion
        where :math:`v_\mathrm{ha}(r)` is the Hartree potential
```

File: atoMEC/staticKS.py [hunk 2]
```diff
@@ -736,8 +750,9 @@ def calc_S_unbound(orbs):
         -----
         Currently only "ideal" (uniform) treatment of unbound electrons is supported.
 
-        .. math:: S_{ub} = \sum_\sigma \frac{V}{\sqrt{2}\pi^2}
-        where :math: 'I_{1/2}' is the complete Fermi-Dirac integral of order 1/2
+        .. math::
+            S_{ub} = \sum_\sigma \frac{V}{\sqrt{2}\pi^2} I_{1/2}(\mu,\beta)
```
Comment (DanielKotik) [inline atoMEC/staticKS.py:754]: Change to
```suggestion
            S_\mathrm{ub} = \sum_\sigma \frac{V}{\sqrt{2}\pi^2} I_{1/2}(\mu,\beta)
```
in order to conform with `S_\mathrm{b}` from line 706.

File: atoMEC/staticKS.py [hunk 3]
```diff
@@ -420,10 +420,14 @@ def calc_v_ha(density, xgrid):
         Notes
         -----
         v_ha is defined on the r-grid as:
```
Comment (DanielKotik) [inline atoMEC/staticKS.py:422]:
```suggestion
        :math:`v_\mathm{ha}` is defined on the r-grid as:
```

File: atoMEC/staticKS.py [hunk 4]
```diff
@@ -420,10 +420,14 @@ def calc_v_ha(density, xgrid):
...
-        .. math:: v_{ha}(r) = 4\pi*\int_0^r dr' n(r') \frac{r'^2}{\max(r,r')}
+
+        .. math:: 
+            v_\mathrm{ha}(r) = 4\pi*\int_0^r \mathrm{d}r' r'^2 \frac{n(r')}{\max(r,r')}
```
Comment (DanielKotik) [inline atoMEC/staticKS.py:425]:
```suggestion
            v_\mathrm{ha}(r) = 4\pi\int_0^r \mathrm{d}r' r'^2 \frac{n(r')}{\max(r,r')}
```
If `*` shall denote ordinary multiplication, leave it out.

**Review by DanielKotik — CHANGES_REQUESTED** [review]
I've made some suggestions. You can directly apply them if you agree on these.

**Comment by timcallow** [comment]
Looks good, thanks

**Pull request merged**